src/pages/coders.js

```diff
@@ -100,7 +100,11 @@ function Coder({ githubUser, avatar, approved, totalContributions }) {
       {showDetails && (
         <CoderDetails>
           <CoderText>
-            <CoderName>@{githubUser}</CoderName>
+            <CoderName>
+              <a href={`https://github.com/${githubUser}`}
+                target="_blank"
+                style="text-decoration:none">@{githubUser}</a>
```

Comment:
Hi @Koalapvh13, did you try to run the project?

Because this change is invalid and broke the page